### PR TITLE
feat(prompt)!: :sparkles: add prepend breaking change

### LINKED ISSUE #39: [FEAT] automatically prepend `BREAKING CHANGE: `
**Is your feature request related to a problem? Please describe.**
While the extension explicitly prompts the user to enter any breaking changes, it expects you to prepend `BREAKING CHANGE: ` manually. In my opinion this is a cumbersome task that should be automated.

**Describe the solution you'd like**
A possible solution would be to simply prepend `BREAKING CHANGE: ` automatically, if the user writes something when prompted for breaking changes.

**Describe alternatives you've considered**
A simpler, but in my opinion inferior, solution would be to no longer ask for a text value for breaking changes. This could be replaced by asking whether or not there were breaking changes and then adding an exclamation mark to the description when there are.

### PR DESCRIPTION
**Pull request by yi-Xu-0100**
Add a prompt for the breaking news.

BREAKING CHANGE: Add a new prompt for the breaking change before the footer.

Close #39

### CODE REVIEW

**Comment by yi-Xu-0100** [comment]
@vivaxy It will make a breaking change, because it adds a prompt before the footer and if the changes does not know for the user, they may write a footer as the breaking change. It will skip check if they were setting the auto commit option. 😥

Or we can add a window to make a prompt for the breaking change which can be setting never showing next time. 🎈

Or we can add a option about enable the breaking change prompt. ✨

File: src/lib/commit-message.ts [hunk 1]
```diff
@@ -101,10 +110,14 @@ export function serializeHeader(partialCommitMessage: {
 
 export function serialize(commitMessage: CommitMessage) {
   let message = serializeHeader(commitMessage);
-  const { body, footer } = commitMessage;
+  const { body, breakingChange, footer } = commitMessage;
   if (body) {
     message += `\n\n${body}`;
   }
+  if (breakingChange) {
+    message = message.replace(': ', '!: ');
```
Comment (vivaxy) [inline src/lib/commit-message.ts:118]: We can have the `!:` implemented in the `serializeHeader` function, other than `serializeHeader` first and `replace` after.

File: src/lib/prompts.ts [hunk 2]
```diff
@@ -269,7 +280,9 @@ export default async function prompts({
         return false;
       } else if (
         showEditor &&
-        (question.name === 'body' || question.name === 'footer')
+        (question.name === 'body' ||
+          question.name === 'footer' ||
```
Comment (vivaxy) [inline src/lib/prompts.ts:284]: Is this name `breakingChange`?

**Review by vivaxy — CHANGES_REQUESTED** [review]
Thx!

**Comment by yi-Xu-0100** [comment]
Here will redesign after the #99, or they will redesign with together.

The latest change is rebased with the master branch, which may have no change than before. 😅

I will make a request about review if all things are done, please give me more time. 😋

**Pull request closed without merging**